atomic_reactor/core.py

```diff
@@ -214,8 +216,56 @@ def build_image_privileged_container(self, build_image, json_args_path):
         return container_id
 
 
+def retry(function, *args, **kwargs):
```

Comment:
Actual retry functionality is not being tested - coveralls doesn't show `final_excep` line being triggered at all